test/llvmDebug/llvmDebug_test.py

```diff
@@ -0,0 +1,71 @@
+# Python test script to verify the llvm debug info
+# is generated properly during Chapel compilation
```

Comment:
I really like this debug script, but it should be plugged in to our test system so that start_test test/llvmDebug runs it. To do that, you'd rename this script sub_test and make the output match the format start_test is familiar with.

    find test -name sub_test

will show you some examples. It would be just fine to merge this PR before doing this, but if you don't get to it, I'll need to do it once the PR is merged. So, at least it should be mentioned as a TODO in the merge message.